docker-compose/envs/common-blockscout.env

```diff
@@ -79,6 +79,7 @@ DISABLE_READ_API=false
 DISABLE_WRITE_API=false
 DISABLE_INDEXER=false
 DISABLE_REALTIME_INDEXER=false
+DISABLE_TOKEN_INSTANCE_FETCHER=false
```

Comment:
we need a corresponding infra PR to make things explicit or we rely on the default value?

Reply:
I would rely on default values here.